status/types.proto

```diff
@@ -95,6 +95,13 @@ enum CommonFail {
     //  - [**0**] Magic number of the served NeoFS network (big-endian 64-bit
     //  unsigned integer).
     WRONG_MAGIC_NUMBER = 1;
+
+    // [**1026**] Signature verification failure.
+    // Details:
+    //  - [**0**] Reason (big-endian 16-bit unsigned integer):
+    //   - [**0**] Signature does not match.
```

Comment:
Signature is invalid?
Also, what about failing to unmarshal public key?

Reply:
it was decided not to add details since it is not fully clear what format is ok. at least for now

/cc @cthulhu-rider @realloc